_data/versions.yml

```diff
@@ -8,10 +8,26 @@ v3.0:
     ##### Support for etcdv3
 
     - Calico now stores its data in [etcd version 3](https://coreos.com/blog/etcd3-a-new-etcd.html).
-    
+
+    > **Important**: If using the etcd datastore it is a requirement to use v3,
+    > and if upgrading a previous deployment it is necessary to migrate the
+    > data as part of the [upgrade process](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/)
+    {: .alert .alert-danger}
+
     ##### Migration and upgrade from v2.6.4
     
     - This version of Calico supports [migration and upgrade](/v3.0/getting-started/kubernetes/upgrade/) from Calico v2.6.4. 
+
+    - The Calico API has changed, which requires that Calico manifests, loaded using the `calicoctl` command, must be updated or converted to the new API.
```

Comment:
Maybe "Calico resource model has been updated" would be clearer than "Calico API has changed"?  (People might not think of calicoctl as an API.)

Reply:
I like "resource model", sounds good